### ISSUE TITLE
Detection of parameters and variables with line breaks

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSRule.Rules.Azure#811]
**Description of the issue**

The rules `Azure.Template.UseParameters` and `Azure.Template.UseVariables` don't detect parameters or variables that are used within multiline expressions.

**To Reproduce**

Steps to reproduce the issue:

```json
{
    "name": "frontendEndpoint2",
    "properties": {
        "hostName": "[
            parameters( 'customDomainName' )
        ]",
        "sessionAffinityEnabledState": "Disabled",
        "sessionAffinityTtlSeconds": 0
    }
}
```

**Expected behaviour**

Update rules to avoid these false positive cases.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.4.1**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Microsoft Windows 10.0.19043
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
